cachito/errors.py

```diff
@@ -1,13 +1,13 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 
-class ValidationError(ValueError):
-    """An error was encountered during validation."""
-
-
 class CachitoError(RuntimeError):
     """An error was encountered in Cachito."""
 
 
+class ValidationError(CachitoError, ValueError):
```

Comment:
Interesting approach. This will match either RuntimeError, ValueError, or CachitoError. I think that's ok. We just have to be sure to catch any sort of ValidationError before catching RuntimeError.

Reply:
In hindsight, I would have preferred that all these just inherit from just `Exception` but I didn't want to break compatibility and wanted to have all the exceptions at least inherit from `CachitoError`.